sources/packages/workflow/BPMN/assessment-gateway.bpmn

```diff
@@ -336,188 +242,587 @@ createVariablesFromPayload(output.prop("programYear"));</camunda:script>
-    </bpmn:serviceTask>
-    <bpmn:serviceTask id="Activity_0mc2luc" name="Get Token">
-      <bpmn:extensionElements>
-        <camunda:connector>
-          <camunda:inputOutput>
-            <camunda:inputParameter name="url">${keycloakURL}/auth/realms/${realm}/protocol/openid-connect/token</camunda:inputParameter>
-            <camunda:inputParameter name="method">POST</camunda:inputParameter>
-            <camunda:inputParameter name="headers">
-              <camunda:map>
-                <camunda:entry key="Content-Type">application/x-www-form-urlencoded</camunda:entry>
-                <camunda:entry key="Authorization">Basic ${base64userpass}</camunda:entry>
-                <camunda:entry key="Accept">application/json</camunda:entry>
-              </camunda:map>
-            </camunda:inputParameter>
-            <camunda:inputParameter name="payload">grant_type=client_credentials</camunda:inputParameter>
-            <camunda:outputParameter name="simsApiAccessToken">${S(response).prop("access_token").stringValue()}</camunda:outputParameter>
-          </camunda:inputOutput>
-          <camunda:connectorId>http-connector</camunda:connectorId>
-        </camunda:connector>
-      </bpmn:extensionElements>
-      <bpmn:incoming>Flow_0iwnsaf</bpmn:incoming>
-      <bpmn:outgoing>SequenceFlow_1o2sazf</bpmn:outgoing>
+      <bpmn:incoming>Flow_0jl2fu1</bpmn:incoming>
+      <bpmn:incoming>SequenceFlow_03x2l2k</bpmn:incoming>
+      <bpmn:outgoing>Flow_1xe24er</bpmn:outgoing>
     </bpmn:serviceTask>
-    <bpmn:intermediateCatchEvent id="Event_1wxl0x4" name="ready-for-assessment">
-      <bpmn:outgoing>Flow_0iwnsaf</bpmn:outgoing>
-      <bpmn:linkEventDefinition id="LinkEventDefinition_1u9ocgm" name="ready-for-assessment" />
+    <bpmn:exclusiveGateway id="Get_Assessment_Error_Gateway">
+      <bpmn:incoming>Flow_19n0g7g</bpmn:incoming>
+      <bpmn:outgoing>Flow_0jl2fu1</bpmn:outgoing>
+      <bpmn:outgoing>Flow_0v56rad</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateCatchEvent id="Event_0halwwn" name="Get Application Timer">
+      <bpmn:incoming>Flow_09vg46t</bpmn:incoming>
+      <bpmn:outgoing>Flow_19n0g7g</bpmn:outgoing>
+      <bpmn:timerEventDefinition id="TimerEventDefinition_08ha5ra">
+        <bpmn:timeDuration xsi:type="bpmn:tFormalExpression">PT1M</bpmn:timeDuration>
+      </bpmn:timerEventDefinition>
     </bpmn:intermediateCatchEvent>
-    <bpmn:endEvent id="Event_0wu28r4">
-      <bpmn:incoming>Flow_1gq6278</bpmn:incoming>
+    <bpmn:intermediateThrowEvent id="Event_0gpagjg" name="Notify error">
+      <bpmn:extensionElements />
+      <bpmn:incoming>Flow_0v56rad</bpmn:incoming>
+      <bpmn:outgoing>Flow_1dd2smr</bpmn:outgoing>
+      <bpmn:messageEventDefinition id="MessageEventDefinition_0os8qcn">
+        <bpmn:extensionElements>
+          <camunda:connector>
+            <camunda:inputOutput>
+              <camunda:inputParameter name="to">gurumoorthy.mohan@gov.bc.ca</camunda:inputParameter>
```

Comment:
not sure, if this is part of your PR. just a thought, we can add this email address as a dynamic variable right

Reply:
Yes, we can, but not part of the PR. The entire error handling part must be changed.